x-pack/plugins/security_solution/public/cloud_security_posture/components/entity_insight.tsx

```diff
@@ -36,62 +31,23 @@ export const EntityInsight = <T,>({
   const { euiTheme } = useEuiTheme();
   const insightContent: React.ReactElement[] = [];
 
-  const { data: dataMisconfiguration } = useMisconfigurationPreview({
-    query: buildEntityFlyoutPreviewQuery(fieldName, name),
-    sort: [],
-    enabled: true,
-    pageSize: 1,
-  });
-
-  const passedFindings = dataMisconfiguration?.count.passed || 0;
-  const failedFindings = dataMisconfiguration?.count.failed || 0;
-
-  const hasMisconfigurationFindings = passedFindings > 0 || failedFindings > 0;
-
-  const { data } = useVulnerabilitiesPreview({
-    query: buildEntityFlyoutPreviewQuery(fieldName, name),
-    sort: [],
-    enabled: true,
-    pageSize: 1,
-  });
```

Comment:
it seems we didn't use `ignore_unavailable` here, but we'll do so now with the hook. How will that affect the behavior?

Reply:
useVulnerabilitiesPreview doesnt use the ignore_unavailable field so it should have no effect